xdis/__init__.py

```diff
@@ -119,6 +119,7 @@
     CO_FUTURE_BARRY_AS_DBFL,
     CO_FUTURE_DIVISION,
     CO_FUTURE_GENERATOR_STOP,
+    CO_FUTURE_ANNOTATIONS,
```

Comment:
Previously these lines were in alphabetic order. Preserving that would be nice.

Reply:
oh, thanks! I was thinking like a minute deducing what was the order about...  but didn't guess it :)